recipes/guile/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "v2.1.2" %}
+
+package:
+  name: guile
+  version: {{ version }}
+
+source:
+  git_url: http://git.savannah.gnu.org/r/guile.git
+  git_tag: f9490cdb3fd9c99de6976077cb87ef77c591967a
```

Comment:
I don't believe this is considered stable. Could we please use 2.0.11, which is stable? For that one, we can use this link ( https://ftp.gnu.org/pub/gnu/guile/guile-2.0.11.tar.gz ) and thus avoid cloning this quite large (~73MB) repo. Please make sure to include a checksum with the file download.